src/apps/main/remote-sync/handlers.ts

```diff
@@ -77,14 +80,27 @@ ipcMain.handle('get-remote-sync-status', () =>
   remoteSyncManager.getSyncStatus()
 );
 
+export async function updateRemoteSync(): Promise<void> {
+  await sleep(2_000);
+  await remoteSyncManager.startRemoteSync();
+  updateSyncEngine();
+}
+export async function fallbackRemoteSync(): Promise<void> {
+  await sleep(2_000);
+  fallbackSyncEngine();
+}
+
+ipcMain.handle('SYNC_MANUALLY', async () => {
+  Logger.info('[Manual Sync] Received manual sync event');
+  await updateRemoteSync();
+  await fallbackRemoteSync();
+});
+
 eventBus.on('RECEIVED_REMOTE_CHANGES', async () => {
   // Wait before checking for updates, could be possible
```

Comment:
Those comments do not apply to the handler but to the updateRemoteSync function now. We could move them to the function or simply remove it